packages/sdk/schema/src/projection.ts

```diff
@@ -112,9 +116,90 @@ export class ViewProjection {
     return { field, props };
   }
 
-  /** Get all field projections */
+  /**
+   * Get projection of View fields and JSON schema property annotations, returning undefined if field doesn't exist.
+   * @param fieldId The ID of the field to get projection for
+   */
+  tryGetFieldProjection(fieldId: string): FieldProjection | undefined {
+    invariant(this._schema.jsonSchema.properties);
+    const field = this._view.fields.find((field) => field.id === fieldId);
+    if (!field) {
+      return undefined;
+    }
+
+    return this.getFieldProjection(fieldId);
+  }
+
+  private _fieldProjectionsComputed = computed(() =>
```

Comment:
I'd put this at the top of the class with the rest of the class properties I think